pkgs/applications/editors/emacs/generic.nix

```diff
@@ -58,16 +58,26 @@ let emacs = stdenv.mkDerivation (lib.optionalAttrs nativeComp {
 
   enableParallelBuilding = true;
 
+  patches = patches ++ (
+    lib.optional (stdenv.isDarwin && stdenv.isAarch64)
+      [
```

Comment:
These square brackets are not needed:
`lib.optional True x = [ x ]`
`lib.optional True [ x ] = [ [ x ] ]`
`lib.optionals True [ x y ] = [ x y ]`